packages/stores/src/storeProvider.tsx

```diff
@@ -28,9 +28,7 @@ const StoreProvider = ({ children, store }: React.PropsWithChildren<{ store: TCo
 
     return (
         <StoreContext.Provider value={memoizedValue}>
-            <WebsiteStatusProvider>
```

Comment:
@adrienne-deriv Could we instead comment out the use of `useSubscription` in the `WebsiteStatusProvider` and replace it with `useFetch`? So we could have the `const { website_status } = useStore()` and the legacy subscription at the same time.